docs/faq.md

```diff
@@ -0,0 +1,148 @@
+
+### How does meshcloud GmbH secure the Infrastructure for my meshStack?
+
+The various cloud components used by meshStack, such as Virtual Cloud Network, Network Load Balancers, and Ingress Controller (Kubernetes) are configured according to best practices.
+
+Kubernetes nodes are secured by a shared responsibility model. meshcloud GmbH uses best-practice configurations & network policies to secure the nodes for your meshStack. Access to these Kubernetes nodes is possible through the project/role assignment of the respective cloud provider.
+
+- [AWS EKS security overview](https://docs.aws.amazon.com/eks/latest/userguide/security.html)
+- [GCP GKE security overview](https://cloud.google.com/kubernetes-engine/docs/concepts/security-overview)
+- [Azure AKS security overview](https://docs.microsoft.com/en-us/azure/aks/concepts-security)
+
+Technical safeguards are implemented such as detecting and responding to Distributed Denial-of-Service (DDoS) attacks, data encryption, etc by leveraging cloud provider services.
+
+A dedicated Virtual Cloud Network is provided for the meshStack. This network is completely encapsulated from other meshStack and their dedicated customer networks. The network configuration also includes the usage of Security Groups to strictly configure communication in the network.
+
+### Does meshStack support Single sign-on (SSO) and Multi-Factor-Authentication (MFA)?
+
+Yes, meshStack supports SSO and MFA.
+
+Activating SSO requires the integration between your own Enterprise Identity Provider (IDP) (e.g. Azure AD) with your respective meshStack.
+
+MFA must be activated in the Enterprise IDP which is integrated with your meshStack.
+
+### Are we allowed to security test meshStack on our own?
+
+Yes, you can perform security tests like Penetration Tests on meshStack but this always requires prior information and getting signed approval from meshcloud GmbH.
+
+### Can meshcloud GmbH employees (aka meshis) access our organizational data?
+
+No, meshis (employee at meshcloud) do not primarily have any access to your organizational data. Though, in certain scenarios such as for support cases if you would like to give the access or permission to your cloud environment then you can assign it to any of our relevant meshi's.
+
+## Data Protection
+
+### Does meshStack process any Personal Identifiable Information (PII)?
+
+meshStack stores your *First Name*, *Last Name*, *Email Address*, and *Last Login Date*. All these are stored in our database for user management (which is also managed via “a locally managed instance of ***[Keycloak](https://www.keycloak.org/)”***).
+
+The External User Identifier (like the Email Address) is replicated to the respective integrated cloud platforms, e.g. AWS, GCP, Azure, etc.
```

Comment:
The External User Identifier (EUID), usually the email address, is replicated to the respective integrated cloud platforms, e.g. AWS, GCP, Azure, etc.